.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -10,7 +10,7 @@ pipeline {
     }
 
     options {
-        timeout(time: 3, unit: 'HOURS')
+        timeout(time: 10, unit: 'HOURS')
```

Comment:
10 hours o_O ?

Why here are enough 3 hours ? https://github.com/kiegroup/kogito-serverless-operator/pull/95/files#diff-3cd97182ce32c1e07b38d60b675724f8b034c07c0154921804b14fcb414eedb8R14

Reply:
Simply because there is a lock in OCP testing: https://github.com/kiegroup/kogito-serverless-operator/pull/95/files#diff-3cd97182ce32c1e07b38d60b675724f8b034c07c0154921804b14fcb414eedb8R172
and we could wait for hours before having access to the cluster to perform the tests.
Thus a big timeout